### PR TITLE
Umass mercantile

### PR DESCRIPTION
**Pull request by adpjay**
Once/if these changes get in, I'll go back and refactor LynnTech and Atrius to use the MyChartAPI functions to reduce code duplication.

### CODE REVIEW

File: MyChart.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
+const sites = require('../data/sites.json');
```
Comment (livgust) [inline MyChart.js:1]: I don't get why this file is named MyChart but the file indicates that it's for Lynn Tech?

**Review by livgust — CHANGES_REQUESTED** [review]
One small comment - I haven't tested yet since there are integration conflicts. Mind resolving and sending back? Thanks!

File: MyChart.js (hunk 1 above)
Comment (adpjay) [reply MyChart.js:1]: I changed the error line referring to Lynn Tech. Refactoring error.

**Comment by livgust** [comment]
@adpjay when you get a chance can you confirm that this isn't needed since I approved #39 ?

**Comment by adpjay** [comment]
This work was merged in with the PR#39

**Pull request closed without merging**